serverless.yml

```diff
@@ -86,7 +88,9 @@ custom:
 
   s3:
     host: localhost
-    directory: tmp
+    directory: .s3-local
```

Comment:
The tmp dir is ignored in the .gitignore I think we should just be able to use that but, if not I think you'd wanna add it to gitignore otherwise when you run it seems like it would add to diff locally